.gitignore

```diff
@@ -24,3 +23,5 @@ Gopkg.lock
 *.o
 /src/control/vendor/**/*.os
 *.xml
+utils/certs/daosCA/**
+vos_size.yaml
```

Comment:
Thanks.  Last minute addition before I left

Reply:
@jolivier23 Is there anything you need from me for this to be merged?